File: week2/task4.py
def get_number(index):
    (x,y) = divmod(index,3)
    result = 7*x + 4*y
    print(result)

get_number(1) # print 4 
get_number(5) # print 15 
get_number(10) # print 25 
get_number(30) # print 70

File: week2/task4.js
function getNumber(index){
    var x = Math.floor(index / 3);
    var y = index % 3;
    var result = 7*x + 4*y;
    console.log(result);// your code here
}
getNumber(1); // print 4
getNumber(5); // print 15
getNumber(10); // print 25
getNumber(30); // print 70